syncall/tw_caldav_utils.py

```diff
@@ -107,7 +115,15 @@ def convert_caldav_to_tw(caldav_item: Item) -> Item:
         tw_item["due"] = caldav_item["due"]
 
     if "categories" in caldav_item.keys():
-        tw_item["tags"] = caldav_item["categories"]
+        categories = caldav_item["categories"]
+        # Search the first 'proj:tag' and use it for the project
+        # Then remove it from the list
+        for i in categories:
+            if i.startswith("proj:"):
+                tw_item["project"] = i.split(":")[1].strip()
+                categories.remove(i)
```

Comment:
A few things to note:

- Let's use `category` instead of `i` during the loop - it makes it clearer as to what the loop variable really is.
- Removing from a list while iterating is potentially dangerous - see posts like this: https://stackoverflow.com/questions/1207406/how-to-remove-items-from-a-list-while-iterating

Could we instead use the following:

```
    if "categories" in caldav_item.keys():
        for category in caldav_item["categories"]:
            if category.startswith("proj:"):
                tw_item["project"] = category.split(":")[1].strip()
            else:
                tw_item["tags"].append(category)
```

As with the earlier snippet, I have not tested it.
Also note that this behavior is slightly different since, if there are mulitple projects by mistake specified by the user of caldav, then this will use the last one specified but I think this is a valid alternative anyway.